common/datasets/librispeech.py

```diff
@@ -89,7 +89,7 @@ def get_bliss_corpus_dict(audio_format="flac", subdir_prefix=""):
     subdir_prefix = (
         os.path.join(subdir_prefix, "LibriSpeech")
         if subdir_prefix
-        else None
+        else ""
```

Comment:
Both is wrong, this if clause is not needed at all now that string is the default ;)

Reply:
If we discard the addition of `"LibriSpeech"` yes. So should I remove it?

Reply:
Or do you want to have it in any case?

Reply:
I think it's nicer in any case, see the force-pushed commit.

Reply:
Yes this is exactly what i meant, removing the condition and just adding it per default